documentation/docs/quickstart/index.md

```diff
@@ -489,6 +489,6 @@ For more information, see [PMM Dashboards](../use/dashboards-panels/index.md).
 ## Next steps
 
 - [Configure PMM via the interface](../configure-pmm/configure.md)
-- [Manage users in PMM](../how-to/manage-users.md)
+- [Manage users in PMM](../pmm-admin/manage-users/index.md)
```

Comment:
pmm-admin in url confuses me like it's related to pmm-admin tool we have. maybe it should be just "admin" or "administration"

Reply:
ok, I've renamed the folder. Thanks!